commits.go

```diff
@@ -546,3 +547,47 @@ func (s *CommitsService) RevertCommit(pid interface{}, sha string, opt *RevertCo
 
 	return c, resp, err
 }
+
+// GPGSubKey represents a GPG subkey ID.
+//
+// GitLab API docs: https://docs.gitlab.com/ee/api/commits.html#get-gpg-signature-of-a-commit
+type GPGSubKey struct {
+	KeyID sql.NullInt64
```

Comment:
This is not something I would like to add. Can you explain why you used `sql.NullInt64`?